src/features/moviesPage/moviePage/BigMovieTile/index.js

```diff
@@ -0,0 +1,23 @@
+import { imagesApi } from "../../../../core/API";
+import { BackDropWrapper, BackGround, Backdrop, Information, Rate, Rating, Star, Title, Votes } from "./styled";
+import star from "../../../../common/images/star.svg"
+
+const BigMovieTile = ({ poster, title, rate, voteCount }) => (
```

Comment:
Names need to be worked on. 
Maybe a moviePageHeader would be better, You could then give it a header tag and it would be quite SEO compatible